destination/destination_integration_test.go

```diff
@@ -0,0 +1,339 @@
+// Copyright © 2022 Meroxa, Inc. & Yalantis
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package destination
+
+import (
+	"context"
+	"crypto/rand"
+	"encoding/hex"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/jmoiron/sqlx"
+	"github.com/matryer/is"
+
+	"github.com/conduitio-labs/conduit-connector-clickhouse/config"
+)
+
+func TestDestination_Write(t *testing.T) {
```

Comment:
WDYT about implementing the "write part" of the acceptance test now? Or maybe that would be too much custom code for reading the records?

Reply:
I think it's not necessary until we implement the source because only the `TestDestination_Write_Success` method checks the destination, but it needs the source.